controller/setting_controller.go

```diff
@@ -1258,12 +1258,12 @@ func (sc *SettingController) updateInstanceManagerCPURequest(dataEngine longhorn
 		return nil
 	}
 
-	detached, _, err := sc.ds.AreAllVolumesDetached(dataEngine)
+	stopped, _, err := sc.ds.AreAllEngineInstancesStopped(dataEngine)
 	if err != nil {
-		return errors.Wrapf(err, "failed to check volume detachment for %v setting update", settingName)
+		return errors.Wrapf(err, "failed to check engine instances for %v setting update", settingName)
 	}
-	if !detached {
-		return &types.ErrorInvalidState{Reason: fmt.Sprintf("failed to apply %v setting to Longhorn components when there are attached volumes. It will be eventually applied", settingName)}
+	if !stopped {
+		return &types.ErrorInvalidState{Reason: fmt.Sprintf("failed to apply %v setting to Longhorn components when there are running engine instances. It will be eventually applied", settingName)}
 	}
```

Comment:
Any idea why we use `AreAllEngineInstancesStopped` instead of `AreAllVolumesDetachedState` ? Any convention around this choice?

Reply:
Same question.

Reply:
Thanks @james-munson